pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md

```diff
@@ -217,12 +218,29 @@ To use the virtual IP, add these flags to the `create cluster` command:
 
 ### Virtual IP Example
 
+Example
+If we have the following networking configurations on the control plane nodes:
+
+```bash
+Control plane node 1:
+ eth0: 1.2.3.4/29
+ eth1: 10.1.2.1/24
+Control plane node 2:
+ eth0: 5.6.7.8/25
+ eth1: 10.1.2.2/24
+Control plane node 3:
+ eth0: 9.10.11.12/30
+ eth1: 10.1.2.3/24
+```
+Then the following command should look like: 
+
 ```bash
 dkp create cluster preprovisioned \
     --cluster-name ${CLUSTER_NAME} \
-    --control-plane-endpoint-host 196.168.1.10 \
+    --control-plane-endpoint-host 10.1.2.0 \
```

Comment:
I think it would be clearer to use an IP in the middle of the CIDR range rather than the first address.  

```suggestion
    --control-plane-endpoint-host 10.1.2.5 \
```